eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/AssemblyCheckerProcessorTest.java

```diff
@@ -186,4 +189,10 @@ public void validReferenceSeqNameInvalidCoordinates() throws Exception {
         assertFalse(processorSeqName.process(input).isAssemblyMatch());
     }
 
+    @Test
+    public void softMaskingShouldNotMatter() throws Exception {
```

Comment:
I was using the semantic vocabulary instead of describing sintactic behaviour, which is usually recommended. you can google what soft-masking is, but if we use the words case-insensitive no place in the code will give a clue on why that happens